package.json

```diff
@@ -1,39 +1,42 @@
 {
   "name": "debugger",
-  "main": "./lib/main",
-  "author": "philippvr",
+  "main": "./lib/index",
+  "author": "AtomDebugger",
   "version": "0.0.0",
   "license": "MIT",
+  "private": true,
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": ">=1.9.0 <2.0.0"
   },
+  "scripts": {
+    "test": "(apm test) && (flow check | grep -q '0 errors') && (eslint . )"
```

Comment:
This will hide any errors from `flow check`, just remove the `grep -q '0 errors'` part and you will get the output. `flow` already exits with a non-zero code (`2`) if it finds issues so you aren't gaining anything with passing it into `grep`.